pkg/api/errors/pubsub.go

```diff
@@ -150,7 +150,7 @@ func (p *PubSubMetadataError) TopicEmpty() error {
 	return p.build(
 		codes.InvalidArgument,
 		http.StatusNotFound,
-		fmt.Sprintf("topic is empty in pubsub %s", p.p.name),
+		"topic is empty in pubsub "+p.p.name,
```

Comment:
```suggestion
		"topic is empty in pubsub " + p.p.name,
```
I'm surprised gofmt didn't complain about this

Reply:
TIL: gofmt doesn't force spaces around operators if the expression is an argument of a function with more than one argument 🤷‍♀️ 
Ref: https://github.com/golang/go/issues/34426, https://github.com/golang/go/issues/12105